nix/project.nix

```diff
@@ -91,6 +94,9 @@ let
             plutus-tx-plugin.ghcOptions = [ "-Werror" ];
           };
         }
+        ({ lib, pkgs, ... }: lib.mkIf (pkgs.stdenv.hostPlatform.isWindows) {
+          packages.basement.configureFlags = [ "--hsc2hs-option=--cflag=-Wno-int-conversion" ];
```

Comment:
Is there a problem with int-conversion on Windows? Would be nice to have a comment explaining this special case with a reference to an issue if applicable.

Reply:
This fixes this https://ci.iog.io/build/8529222/nixlog/1error:
```
Preprocessing library for basement-0.0.16...
Size.hsc:126:30: error: initialization of ‘long long int’ from ‘void *’ makes integer from pointer without a cast []
compilation failed
```

`basement` repo was archived, so there's no way to open a ticket.  Maybe you'll need to consider moving away of it.

Reply:
This is how we inherit `basement`:
```
 plutus-core-1.49.0.0
   └─ cardano-crypto-class-2.2.0.0
       └─ crypton-1.0.1
           └─ memory-0.18.0
               └─ basement-0.0.16
  ```
The newest `cardano-crypto-class` depends on `crypton` https://chap.intersectmbo.org/package/cardano-crypto-class-2.2.2.0/

The newest `crypton` depends on `basement`:
https://flora.pm/packages/@hackage/crypton/1.0.4/dependencies

@coot please add the explanation you gave in the github comment above as a code comment in the `nix` file, where the flags are set.

Reply:
I opened an issue for it: kazu-yamamoto/crypton#57